convention_about_external_functions.txt

```diff
@@ -248,3 +248,45 @@ outputs:
 1) [jac_x]      size: nx*nx     type: BLASFEO_DMAT
 2) [jac_xdot]   size: nx*nx     type: BLASFEO_DMAT
 3) [jac_u]      size: nx*nu     type: BLASFEO_DMAT
+
+/************************************************
+* sim_gnsf
+************************************************/
+
+/* nonlinearity function phi */
+
+external functions: phi_fun, phi_fun_jac_y, phi_jac_y_uhat
```

Comment:
I would split these functions, and have separate inputs and outputs for each of them; I think it makes it more clear